app/routes/contact/components/ContactForm.js

```diff
@@ -57,14 +57,22 @@ const ContactForm = (props: Props) => {
     <Form onSubmit={props.handleSubmit(submit)}>
       <p>
         Dette skjemaet kan benyttes for å kontakte Abakus sine komiteer eller
-        Hovedstyret. Sender du til en spesifikk komité er det kun lederen av
+        Hovedstyret. Dersom du har opplevd avvik eller ubehagelige hendelser i
+        forbindelse med Abakus, ønsker vi gjerne at dette meldes fra om her.
+        Send oss gjerne også spørsmål, feedback eller ting du ønsker å fortelle
+        oss. Du kan også benytte dette skjemaet om du ønsker å avtale tid for
+        innsyn i HSP-avtalen.
+      </p>
+      <p>
+        Sender du meldingen til en spesifikk komité er det kun lederen av
         komiteen som vil motta meldingen. Om du sender til Hovedstyret vil hele
-        styret få meldingen.
+        styret motta meldingen.
       </p>
       <p>
-        Send oss gjerne spørsmål, feedback eller ting du ønsker å fortelle oss.
-        Du kan også benytte dette skjemaet om du ønsker å avtale tid for innsyn
-        i HSP-avtalen.
+        Skjemaet kan både benyttes til å sende inn en anonym henvendelse, eller
+        en med avsender for videre oppfølging. Både komitéledere og Hovedstyret
+        som mottar henvendelser har signert taushetserklæring, og vil følge opp
+        saken dersom det ønskes.
```

Comment:
> og vil følge opp saken dersom det ønskes.

hmm, should we add what we mean by this? Will they contact anyone else etc. etc. Or, besides, does it not go without saying? Again, might be overthinking, but I'm simply trying to picture myself in the shoes of someone in desperate need of help 😅

Reply:
I feel like it's nice to state explicitly that the case will be followed up on, so there's no doubt about it. It's probably very uncomfortable to be unsure of whether or not your message will lead to any action if you've just reported an uncomfortable situation. But I can change it to "og vil kontakte deg og følge opp saken dersom det ønskes." so they know that when someone sees the message, they will get in contact.

Reply:
Yes, the new suggestion is good! I think it's nice to "know" for certain that you will be contacted, sort of assuring in a way.